Revan Sarbast/css/style.css

```diff
@@ -0,0 +1,167 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+html,body{
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(238, 238, 241);
+}
+header{
+    width: 100%;
+    height: 70px;
+    display: flex;
```

Comment:
we didn't really want you to use inline-block and maybe some floats since we didn't get to flex yet, but it's alright for now